grpcbp/prometheus.go

```diff
@@ -1,19 +1,21 @@
 package grpcbp
 
 import (
+	"strings"
+
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 
 	"github.com/reddit/baseplate.go/prometheusbp"
 )
 
 const (
-	localServiceLabel      = "grpc_service"
-	methodLabel            = "grpc_method"
-	typeLabel              = "grpc_type"
-	successLabel           = "grpc_success"
-	codeLabel              = "grpc_code"
-	remoteServiceSlugLabel = "grpc_slug"
+	serviceLabel          = "grpc_service"
+	methodLabel           = "grpc_method"
+	typeLabel             = "grpc_type"
+	successLabel          = "grpc_success"
+	codeLabel             = "grpc_code"
+	remoteServerSlugLabel = "grpc_slug"
```

Comment:
we originally added `remote-` prefix to this name because we thought the other label (`grpc_service`) is for local. but now we know both are for the remote, the `remote-` prefix in the name doesn't make much sense any more, and I'd suggest just rename this const to `serverSlugLabel` or even just `slugLabel`.

Reply:
i have a slight preference to `serverSlugLabel`. Will make that change.